nautobot/extras/templates/extras/inc/jobresult.html

```diff
@@ -38,7 +38,7 @@
 </div>
 {% plugin_left_page result %}
 <div class="col-md-6">
-{% include 'inc/custom_fields_panel.html' %}
+{% include 'inc/custom_fields_panel.html' with custom_fields=object.get_custom_fields %}
```

Comment:
Ew, I guess the JobResult detail view is missing an Advanced tab. Mind opening an issue for this in the backlog? If I remember, I'll try to address it as part of the work I'm doing for #1001.

Reply:
https://github.com/nautobot/nautobot/issues/1394